packages/maskbook/src/plugins/ITO/UI/CreateForm.tsx

```diff
@@ -0,0 +1,325 @@
+import { createStyles, makeStyles, Box, TextField, Grid } from '@material-ui/core'
+import { useState, useCallback, useMemo, useEffect, ChangeEvent } from 'react'
+import { useStylesExtends } from '../../../components/custom-ui-helper'
+import { EthereumStatusBar } from '../../../web3/UI/EthereumStatusBar'
+import { useI18N } from '../../../utils/i18n-next-ui'
+import { EthereumTokenType } from '../../../web3/types'
+import { useAccount } from '../../../web3/hooks/useAccount'
+import { useConstant } from '../../../web3/hooks/useConstant'
+import { ITO_CONSTANTS } from '../constants'
+import { ApproveState, useERC20TokenApproveCallback } from '../../../web3/hooks/useERC20TokenApproveCallback'
+import { ExchangeTokenPanelGroup } from './ExchangeTokenPanel'
+import { useCurrentIdentity } from '../../../components/DataSource/useActivatedUI'
+import BigNumber from 'bignumber.js'
+import type { PoolSettings } from '../hooks/useFillCallback'
+import 'date-fns'
+import type { ExchangeTokenAndAmountState } from '../api/useExchangeTokenAmountstate'
+import { useTokenBalance } from '../../../web3/hooks/useTokenBalance'
+import ActionButton from '../../../extension/options-page/DashboardComponents/ActionButton'
+import { v4 as uuid } from 'uuid'
+import { useChainIdValid } from '../../../web3/hooks/useChainState'
+
+const useStyles = makeStyles((theme) =>
+    createStyles({
+        line: {
+            margin: theme.spacing(1),
+            display: 'flex',
+        },
+        flow: {
+            margin: theme.spacing(1),
+            textAlign: 'center',
+        },
+        bar: {
+            padding: theme.spacing(0, 2, 2),
+        },
+        input: {
+            padding: theme.spacing(1),
+            flex: 1,
+        },
+        label: {
+            paddingLeft: theme.spacing(2),
+        },
+        tip: {
+            fontSize: 12,
+            color: theme.palette.text.secondary,
+        },
+        button: {
+            margin: theme.spacing(2, 0),
+            padding: 12,
+        },
+        date: {
+            flex: 1,
+            padding: theme.spacing(1),
+        },
+    }),
+)
+
+export interface CreateFormProps extends withClasses<KeysInferFromUseStyles<typeof useStyles>> {
```

Comment:
`KeysInferFromUseStyles` has been deprecated. Please don't use it. See the comment on the definition on the `KeysInferFromUseStyles` for details

Reply:
ok